login-workflow/src/screens/EulaScreen/EulaScreen.tsx

```diff
@@ -24,20 +24,24 @@ export const EulaScreen: React.FC<EulaScreenProps> = (props) => {
     const [eulaAccepted, setEulaAccepted] = useState(screenData.Eula.accepted ?? initialCheckboxValue);
     const [isLoading, setIsLoading] = useState(true);
     const [eulaData, setEulaData] = useState<string>();
+    const [isError, setIsError] = useState(false);
```

Comment:
Rename this to something like `eulaFetchError` or track it with a variable called `isEulaLoaded` so it makes more sense syntactically.